recipes/loos/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh -x
+# This install script is intended for conda-forge, and assumes the conda env
+# is already set up. If you need to set up a conda env, I suggest running
+# conda_build.sh instead.
+set -x
+
+#if [ "$platform" = "Linux" ]; then
+#    export CXX=`which g++`
+#elif [ "$platform" = 'Darwin' ]; then
+#    export CXX=`which clang++`
+#else
+#    echo "Unknown platform $platform, assuming g++"
+#    export CXX=`which g++`
+#fi
+echo "Begin loos build.sh"
+which $CXX
+export CONDA_PREFIX=$BUILD_PREFIX
+
+scons PREFIX=$CONDA_PREFIX
```

Comment:
Does `scons` produce a `config.log`?

Reply:
THANK YOU! I'm an idiot for not checking that. Sure enough, it's trying to run gcc and g++ when doing the tests. I'm not sure WHY it's grabbing the wrong compilers (at least the wrong g++, since I verified that CXX is set, but at least this gives me something else to work on. 

I feel pretty stupid right now for not checking this.